src/sasquatchbackpack/fastapi.py

```diff
@@ -1,44 +1,47 @@
 """Backpack FastAPI module."""
 
 from fastapi import FastAPI
+from pydantic import BaseModel
 
 from sasquatchbackpack.sources.usgs.commands import usgs_earthquake_data
 
 app = FastAPI()
 
 
-@app.get(
-    "/sources/usgs/earthquake_data/{days}/{hours}/{radius}/{latitude}/{longitude}/{lower}/{upper}/{publish}/{method}"
-)
-async def read_item(
-    days: int,
-    hours: int,
-    radius: int,
-    latitude: int,
-    longitude: int,
-    lower: int,
-    upper: int,
-    publish: bool,  # noqa: FBT001
-    method: str,
-) -> None:
+class EarthquakeParams(BaseModel):
+    """{days}/{hours}/{radius}/{latitude}/{longitude}/{lower}/{upper}/{publish}/{method}."""
```

Comment:
Since this model represents the request body for querying earthquake events from the USGS API, you could name it `EarthquakeRequest` and also add a nice docstring

```
A request to query Earthquake events from the USGS API.

This model represents the request body for que earthquake query parameters.
```